### ISSUE TITLE
Black and white images when training SRMD

### ISSUE THREAD

**Issue by claroche-gpfw** [cszn/KAIR#30]
Hi,

Thanks for this repository.

I am trying to retrain SRMD x4 on DIV2K so I changed options/train_srmd.json with my database path and changed the frequency of the checkpoints. After one day of training on GPU (around 1300 iterations), all the generated images seem to estimate the HR but in black and white (with 3 channels). I plotted the LR and HR images and they are in colour.

Does the colour usually appear later during the training or is there something to add?

Thanks,
Charles

**Comment by claroche-gpfw** [comment]
Hi,

It seems that the colour appeared during the training but still I cannot reproduce the results of the paper. 
My model seems to have converged (I used a learning rate of 1e-3 for 50 epochs, 1e-4 for the next 100epochs and 1e-5 for the last epochs).
![srmd](https://user-images.githubusercontent.com/65959416/91022911-21ccb200-e5f6-11ea-8512-83d5bfb7be78.png)
However, the model still does not handle well the colour as we can see here:
![sig2 6_img_0_gt](https://user-images.githubusercontent.com/65959416/91023128-7112e280-e5f6-11ea-9ac0-44d6884206dd.png)

I noticed that in comparison to what is mentioned in the paper, you do not use batch norm in the architecture and use L1 loss instead of L2. Do I need to change those parameters to be able to reproduce the training of the pre-trained model you give?

If you have some tips to overcome those colour issues I would be thankful. 

Thanks,
Charles

**Comment by cszn** [comment]
Start the learning rate with 1e-4 if you do not use batch norm.

**Comment by claroche-gpfw** [comment]
Hi, 
I tried and this does not change the results.
